Makefile

```diff
@@ -32,10 +39,10 @@ deploy-init:
 
 deploy-appbundle:
 	cp \
-		$(BUILD_DIR)/app.bundle.min.js \
-		$(BUILD_DIR)/app.bundle.min.map \
-		$(BUILD_DIR)/external_api.min.js \
-		$(BUILD_DIR)/external_api.min.map \
+		$(BUILD_DIR)/app.bundle.js \
+		$(BUILD_DIR)/app.bundle.map \
+		$(BUILD_DIR)/external_api.js \
+		$(BUILD_DIR)/external_api.map \
 		$(OUTPUT_DIR)/analytics.js \
```

Comment:
@damencho I think we should have development and production version of scripts. In this case files are being minimized if we run production building. I got rid of `min` suffix because files are not always being minimized. But I agree that it isn't the cleanest way to separate environments. So, let's discuss it.